code/game/gamemodes/eldritch_cult/eldritch_cult.dm

```diff
@@ -4,7 +4,7 @@
 	report_type = "heresy"
 	antag_flag = ROLE_HERETIC
 	false_report_weight = 5
-	protected_jobs = list("Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain")
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Brig Physician")
```

Comment:
Same as above.

Reply:
ditto

Reply:
Deputy isn't on any protected roles lists for any other antag.

Reply:
Deputy isn't a roundstart job